src/Sulu/Bundle/WebsiteBundle/Sitemap/Provider/PagesSitemapProvider.php

```diff
@@ -30,27 +31,43 @@ class PagesSitemapProvider implements SitemapProviderInterface
     private $contentRepository;
 
     /**
+     * @var WebspaceManagerInterface
+     */
+    private $webspaceManager;
+
+    /**
      * @param ContentRepositoryInterface $contentRepository
      */
-    public function __construct(ContentRepositoryInterface $contentRepository)
-    {
+    public function __construct(
+        ContentRepositoryInterface $contentRepository,
+        WebspaceManagerInterface $webspaceManager
+    ) {
         $this->contentRepository = $contentRepository;
+        $this->webspaceManager = $webspaceManager;
     }
 
     /**
      * {@inheritdoc}
      */
     public function build($page, $portalKey, $locale)
```

Comment:
The `$locale` parameter here is not used anymore, because the provider has to return the `SitemapUrl`s for all locales. We could still leave it like that, then there would be no BC break in this method, and all existing providers would continue to work (although they would not work correctly, because they should return all locales).

So the question is if we just deprecate this parameter, or remove it completely, so that everybody updating has to touch their code (which would be a good idea anyway, because it's not working as it should - so I am favoring this option, although it is a BC break).